lib/admin.php

```diff
@@ -155,6 +161,7 @@ function wc_admin_link_structure() {
 	array_unshift( $submenu['woocommerce'], $menu );
 
 	// Rename "Analytics" to Overview (otherwise this reads Analytics > Analytics).
+	// phpcs:ignore
```

Comment:
This should ignore everything, since we only want to allow overide of a WP global, better use `// WPCS: override ok.` instead.

Reply:
Like: 

```php
$submenu['wc-admin#/analytics'][0][0] = __( 'Overview', 'wc-admin' ); // WPCS: override ok.
```